presto/presto.sh

```diff
@@ -55,7 +55,7 @@ function wait_for_presto_cluster_ready() {
 
 function get_presto(){
   # Download and unpack Presto server
-  wget https://repo1.maven.org/maven2/com/facebook/presto/presto-server/${PRESTO_VERSION}/presto-server-${PRESTO_VERSION}.tar.gz
+  wget https://repo1.maven.org/maven2/io/prestosql/presto-server/${PRESTO_VERSION}/presto-server-${PRESTO_VERSION}.tar.gz
```

Comment:
Can you provide more information about io.prestosql? e.g., project homepage. How is it different from com.facebook.presto?

Reply:
@functicons, please see @martint's (one of the creators of presto) explanation: https://github.com/prestosql/presto/issues/380#issuecomment-469525303